src/util/vector.ts

```diff
@@ -2,3 +2,19 @@ export interface Vector2 {
   x: number;
   y: number;
 }
+
+export function areEqual(v1: Vector2, v2: Vector2): boolean {
```

Comment:
Think this is cruft left from a merge commit. Vector math is now in the math package.